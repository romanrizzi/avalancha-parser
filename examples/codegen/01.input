
print A
print B
print True
print False
print Cons(Zero, Cons(Suc(Zero), Cons(Suc(Suc(Zero)), Nil)))

