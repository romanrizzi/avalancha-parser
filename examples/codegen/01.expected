A
B
True
False
Cons(Zero, Cons(Suc(Zero), Cons(Suc(Suc(Zero)), Nil)))
